pkg/clients/helm/client.go

```diff
@@ -254,7 +255,7 @@ func (hc *client) PullAndLoadChart(spec *v1beta1.ChartSpec, creds *RepoCreds) (*
 	var chartFilePath string
 	var err error
 	switch {
-	case spec.URL == "" && spec.Version == "":
+	case spec.URL == "" && (spec.Version == "" || spec.Version == devel):
```

Comment:
Hmm, what if user sets the version as `>0.0.0` and expect to have the latest _stable_ release run? 
I am wondering if the previous PR explicitly setting devel flag would be a better approach 🤔

Reply:
The previous PR doesn't work.
Apparently, the Helm client doesn't recognize the devel flag. 
That's the reason why we need to explicit provide this version.

Reply:
Thanks, sounds reasonable. 

Could you check CI, it is failing.

Reply:
Fixed, can you please approve again ?